app/scripts/modules/amazon/src/securityGroup/configure/configSecurityGroup.mixin.controller.js

```diff
@@ -128,10 +128,7 @@ module(AMAZON_SECURITYGROUP_CONFIGURE_CONFIGSECURITYGROUP_MIXIN_CONTROLLER, [
       const account = getAccount();
       const regions = $scope.securityGroup.regions || [];
       VpcReader.listVpcs().then(function(vpcs) {
-        const vpcsByName = _.groupBy(
-          vpcs.filter(vpc => vpc.account === account),
-          'label',
-        );
+        const vpcsByName = _.groupBy((vpcs || []).filter(vpc => vpc.account === account), 'label');
```

Comment:
Was `vpcs` actually null? How does this happen?

Reply:
I could not reproduce locally, so I am not sure how this happens. But on main, if you toggle regions (which calls this function) it will through an error `cannot read property filter of undefined`

Reply:
I think this is happening because we did not cut a new release of the AWS module after [this](https://github.com/spinnaker/deck/commit/2b6a9411bfe3d5fd54d815065fe9d99625747241) PR was merged.

Reply:
Yes, let me bump aws then we can see if this still happens